src/Stormpath/Authc/Api/BasicAuthenticationResult.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Stormpath\Authc\Api;
+
+
+class BasicAuthenticationResult extends AuthenticatorResult
+{
+
+}
```

Comment:
The class is just a wrapper of AuthenticatorResult so we get the correct return type per the spec.